src/dune

```diff
@@ -47,6 +47,14 @@
       (apron -> relationPrecCompareUtil.apron.ml)
       (-> relationPrecCompareUtil.no-apron.ml)
     )
+    (select relationalStringAnalyses.ml from
+      (apron -> relationalStringAnalyses.apron.ml)
+      (-> relationalStringAnalyses.no-apron.ml)
+    )
+    (select relationalStringDomains.ml from
+      (apron -> relationalStringDomains.apron.ml)
+      (-> relationalStringDomains.no-apron.ml)
+    )
```

Comment:
The fact that a relational string analysis, which has nothing to do with Apron, needs to do these hacks, means that we really need to refactor our generic relational analysis first to decouple it from Apron.